TestVectorsAwsCryptographicMaterialProviders/runtimes/python/fuzz_generator.py

```diff
@@ -0,0 +1,262 @@
+            "provider-id": f"aws-raw-vectors-persistent-{raw_key_id}",
+            "encryption-algorithm": "aes"
+        }
+    
+    # Create the test vector in the correct format
+    test_vector = {
+        "type": test_type,
+        "description": draw(st.text(min_size=10, max_size=200, alphabet='abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789 ')),  # Normal ASCII
+        "algorithmSuiteId": algorithm_suite,
+        "encryptKeyDescription": key_description,
+        "decryptKeyDescription": key_description,
+        "reproducedEncryptionContext": reproduced_context,
+        "requiredEncryptionContextKeys": required_keys,
+        "encryptionContext": encryption_context
+    }
+    
+    # Add error descriptions for negative tests (NORMAL ASCII)
+    if test_type == "negative-encrypt-keyring":
+        test_vector["errorDescription"] = draw(st.text(min_size=10, max_size=200, alphabet='abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789 '))
+    elif test_type == "negative-decrypt-keyring":
+        test_vector["decryptErrorDescription"] = draw(st.text(min_size=10, max_size=200, alphabet='abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789 '))
+    
+    return test_vector
+
+
+def generate_fuzz_test_vectors(num_vectors: int = 2) -> Dict[str, Any]:
+    """
+    Generate multiple fuzzed test vectors.
+    
+    Args:
+        num_vectors: Number of test vectors to generate
+        
+    Returns:
+        Dictionary containing the test vectors
+    """
+    test_vectors = {}
+    
+    print(f"Generating {num_vectors} fuzzed test vectors...")
+    print("Focus: encryptionContext, requiredEncryptionContextKeys, reproducedEncryptionContext")
+    
+    for i in range(num_vectors):
+        # Use Hypothesis to generate a test vector
+        test_vector = fuzz_test_vector().example()
+        test_id = str(uuid.uuid4())
+        test_vectors[test_id] = test_vector
+        
+        if (i + 1) % 10 == 0:
+            print(f"Generated {i + 1}/{num_vectors} vectors...")
+    
+    return test_vectors
+
+
+def main():
+    """Main function to generate fuzzed test vectors."""
+    print("=== AWS Cryptographic Material Providers Fuzz Testing Generator ===")
+    print("Strategy: Focus on encryption context UTF-8 vulnerabilities")
+    
+    # Generate test vectors
+    test_vectors = generate_fuzz_test_vectors(num_vectors=2)
+    
+    # Create manifest.json with the correct format
+    manifest_data = {
+        "manifest": {
+            "version": 4,
+            "type": "awses-mpl-encrypt"
+        },
+        "keys": "file://keys.json",
+        "tests": test_vectors
+    }
+    
+    # Save to file as manifest.json
+    with open("manifest.json", "w") as f:
+        json.dump(manifest_data, f, indent=2, ensure_ascii=False)
+    
+    print(f"Generated {len(test_vectors)} fuzzed test vectors")
+    print("Saved to manifest.json")
+    
+    # Show a sample of what was generated
```

Comment:
Why is it necessary show a sample in log? Why would someone viewing the CI log care if its passing?